1-getting-started-lessons/1-intro-to-programming-languages/translations/README.nl.md

```diff
@@ -0,0 +1,193 @@
+# Introductie van Programmeertalen en Bijbehorend Vakgereedschap
+
+Deze les beschrijft de basis over programmeertalen. De onderwerpen zijn van toepassing op de meeste hedendaagse programeertalen. In het 'Vakgereedschap' onderdeel leer je over nuttige toepassingen die je helpen als web ontwikkelaar.
+
+![Introductie Programmeren](../webdev101-programming.png)
+> Sketchnote door [Tomomi Imura](https://twitter.com/girlie_mac)
+
+## Pre-Lecture Quiz
+
+[Pre-lecture quiz](../.github/pre-lecture-quiz.md)
+
+## Introductie
+
+In deze les hebben we het over:
+
+- Wat is programmeren?
+- Verschillende typen programmeertalen.
+- Basis elementen van een programma.
+- Nuttige toepassingen en gereedschap voor de professionele web ontwikkelaar.
+
+## Wat is programmeren?
+
+Programmeren (ook wel coderen genoemd) is het process van het schrijven van instructies voor een apparaat, zoals een computer of een mobieltje. We schrijven deze instructies met een programmeertaal, welke wordt geinterpreteerd door het apparaat. Deze sets van instructies staan bekend onder verschillende namen zoals; *programma*, *computer programma*, *applicatie (app)*, en *executable*.
+
```

Comment:
En: script?

Reply:
Done (although the EN text doesn't contain it, but probably should :) )